webpack.base.config.js

```diff
@@ -50,6 +50,7 @@ module.exports = (env) => {
         ], (regex) => { return regex.test(name) })
       },
       rules: [
+        { test: require.resolve('cookieconsent'), use: 'exports-loader?cookieconsent' },
```

Comment:
Just want to clarify my understanding of this fantastic line of code (to improve my webpack understanding).

This rule applies to the npm package `cookieconsent`, and tells webpack to import the library with the `exports-loader` that adds a `module.exports = cookieconsent` to the file?

Very excited by this rule!

Reply:
Yup thats the effective output, the cookie consent module exposes itself on `window` only and using the `exports-loader` allows us to import it like a normal module (with the hope that one day they'll also expose as a commonjs module).